code/game/objects/objs.dm

```diff
@@ -14,6 +14,8 @@
 	var/burn_world_time //What world time the object will burn up completely
 	var/being_shocked = 0
 
+	var/alternate_screams = list() // This is used to add alternate scream sounds to mobs when equipped
```

Comment:
Would be better to define it under items. I don't think we plan on having machinery or structures that alter your screams,do we